composer.json

```diff
@@ -4,12 +4,12 @@
   "license": "MIT",
   "type": "library",
   "require": {
-    "php": "^8.1",
-    "nesbot/carbon": "^2.63"
+    "php": "^8.2",
+    "nesbot/carbon": "^3.5"
   },
   "require-dev": {
-    "blumilksoftware/codestyle": "^1.10",
-    "phpunit/phpunit": "^9.5"
+    "blumilksoftware/codestyle": "^2.8",
```

Comment:
Please update it to 3.1, as this is the newest version.

Reply:
Updated. I took versions from Toby, thats why it was 2.8 not 3.1.